docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -310,6 +310,23 @@ Set one or more OCI labels on the volume. The format is a list of
 
 This key can be listed multiple times.
 
+#### `Device=`
+
+The path of a device which should be mounted for the volume.
+
+#### `Type=`
+
+The filesystem type of `Device` as used by the **mount(8)** commands `-t` flag.
```

Comment:
FWIW, in the man pages, we use `option` rather than `flag`.

```suggestion
The filesystem type of `Device` as used by the **mount(8)** commands `-t` option.
```